output/cloud/output.go

```diff
@@ -259,17 +258,23 @@ func (out *Output) startBackgroundProcesses() {
 	}()
 }
 
-// Stop gracefully stops all metric emission from the output and when all metric
-// samples are emitted, it sends an API to the cloud to finish the test run.
+// Stop is deprecated, see the StopWithTestError() method.
```

Comment:
Just to give a chance for godoc (and :crossed_fingers: at some point vscode) to parse and mark it properly

```suggestion
// Stop gracefully stops all metric emission from the output and when all metric
// samples are emitted, it sends an API to the cloud to finish the test run.
//
// Deprecated: use StopWithTestError() instead.
```

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/62eb4f967a8ac2bb492be65904e12b5d8f312de8..b456ed79672d120a585dda6f066deaa8f8bd5429#diff-f70f77b77caf0bb396844dacff629677ceb27f5e7cd183bca0d7d08b8672e4beR261-R266).